openupgrade_scripts/scripts/crm/15.0.1.6/post-migration.py

```diff
@@ -0,0 +1,17 @@
+from openupgradelib import openupgrade
+
+
+def try_delete_noupdate_records(env):
+    openupgrade.delete_records_safely_by_xml_id(
+        env,
+        [
+            "crm.mail_alias_lead_info",
```

Comment:
Not sure about removing this one...

Reply:
https://github.com/OCA/OpenUpgrade/pull/3504